src/go/k8s/tests/e2e/console-admin-api/04-check-version.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: check-version
+  namespace: console-admin-api
+spec:
+  template:
+    spec:
+      containers:
+        - name: curl
+          image: curlimages/curl:latest
+          env:
+            - name: POD_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+          command:
+            - /bin/sh
+            - -c
+            - -ex
+          args:
+            - >
+              url=http://console.$POD_NAMESPACE.svc.cluster.local:8080/api/cluster
+              res=$(curl --silent -L $url | grep -o '\"kafkaVersion\":[^,}]*' | grep -o '[^:]*$') &&
+              echo $res > /dev/termination-log &&
+              if [[ "$res" != '"Redpanda v'* ]]; then
+                exit 1;
+              fi
```

Comment:
As next test you could create SASL user.

Reply:
When I created this PR the only feature that is enabled by this config is showing the redpanda version. If there is more features now, I don't think we should check all of them here? It's console's domain after all :thinking:

Reply:
I'm not sure if version under the hood reach the Redpanda Admin API.

I had problems using this feature in one of the cluster and I couldn't create SASL user. In the end if console supports it, but operator miss configures it then it's the operator domain.